x-pack/plugins/ml/server/routes/job_validation.ts

```diff
@@ -25,7 +25,9 @@ type CalculateModelMemoryLimitPayload = TypeOf<typeof modelMemoryLimitSchema>;
 /**
  * Routes for job validation
  */
-export function jobValidationRoutes({ config, xpackMainPlugin, router }: RouteInitialization) {
+export function jobValidationRoutes({ version, getLicenseCheckResults, router }: any) {
+  // const { isSecurityDisabled } = getLicenseCheckResults();
```

Comment:
commented code can be removed

Reply:
609be5c4c218d1feed4b249158c438ce7c30f99d